pkg/skaffold/deploy/kustomize.go

```diff
@@ -210,19 +210,7 @@ func (k *KustomizeDeployer) Render(ctx context.Context, out io.Writer, builds []
 		return err
 	}
 
-	manifestOut := out
-	if filepath != "" {
-		f, err := os.OpenFile(filepath, os.O_RDWR|os.O_CREATE, 0666)
-		if err != nil {
-			return fmt.Errorf("opening file for writing manifests: %w", err)
-		}
-		defer f.Close()
-		f.WriteString(manifests.String() + "\n")
-		return nil
-	}
-
-	fmt.Fprintln(manifestOut, manifests.String())
```

Comment:
This change isn't equivalent as when `filepath == ""`, this code would have printed the manifests to `out`.

Reply:
Thanks @briandealwis. In `outputRenderedManifests(renderedManifests string, output string, manifestOut io.Writer)` of `util.go`, if `out == ""` it calls fmt.Fprintln (see Line 104 of `util.go`). So, in this case if `filepath` is "" I think that should do what you're asking doesn't it?

Reply:
My apologies — I could have sworn the `outputRenderedManifests()` was very different.

Reply:
No worries, just glad it does the right thing :)